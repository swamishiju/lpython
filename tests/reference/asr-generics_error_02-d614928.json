{
    "basename": "asr-generics_error_02-d614928",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/generics_error_02.py",
    "infile_hash": "0cd9c07bbe28138a3cca57b750b625c528773ac79c36b15e5dc5830a",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-generics_error_02-d614928.stderr",
    "stderr_hash": "ed8e1b5bb3e56a2855182951d7e800079399fd76e100d4aa9749d611",
    "returncode": 2
}