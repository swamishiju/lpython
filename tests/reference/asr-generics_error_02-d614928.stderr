semantic error: Type mismatch in procedure call; the types must be compatible
  --> tests/errors/generics_error_02.py:37:12
   |
37 | print(mean(["a","b","c"], add=add_string, div=div_string))
   |            ^^^^^^^^^^^^^ type mismatch (passed argument type is list[Allocatable[str]] but required type is list[T])
   |
26 |    def mean(x: list[T], **kwargs) -> f64:
   |    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...
...
   |
35 |        return div(res, k)
   | ...^^^^^^^^^^^^^^^^^^^^^^ type mismatch (passed argument type is list[Allocatable[str]] but required type is list[T])
